docs/DeveloperGuide.adoc

```diff
@@ -721,17 +721,51 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
+|`* * *` | user | add a new person |
+|`* * *` | user | delete a person | remove entries that I no longer need
+|`* * *` | user | find a person by name | locate details of persons without having to go through the entire list
+|`* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
```

Comment:
link does not work. Please sort the stories by relevance, just as it was with the original set.